tools/starman.psgi

```diff
@@ -7,6 +7,8 @@ use LedgerSMB::PSGI;
 use Plack::Builder;
 use Plack::Middleware::Static;
 
+die 'Cannot verify version of libraries, may be including out of date modules?' unless $LedgerSMB::PSGI::VERSION == '1.5';
```

Comment:
@einhverfr any reason not to use the ```$LedgerSMB::VERSION```variable for this? It allows even more exact checking, if we add tools/starman.psgi to the list of files which have a version number encoded; it'll be updated on every release...